js/script.js

```diff
@@ -104,6 +93,71 @@ function createProduct(imgUrl, Name, inStock, price, reviews) {
     reviewsGrade.append(span);
 }
 
+
+
+//* Sort
+
 for (const el of items) {
-    createProduct(el.imgUrl, el.name, el.orderInfo.inStock, el.price, el.orderInfo.reviews)
+    // if (el.color == "Black") {
+    createProduct(el.id, el.imgUrl, el.name, el.orderInfo.inStock, el.price, el.orderInfo.reviews)
+    // }
+}
+
+//* Filter
+
+const filterTitle = document.getElementsByClassName('filter-title')
+
+Array.from(filterTitle, el => el.addEventListener('click', function () {
+    console.log(el)
+    this.querySelector('.filter-icon').classList.toggle('filter-icon-active')
+    this.nextElementSibling.classList.toggle('active')
+    this.parentNode.classList.toggle('filter-card-active')
+}))
+
+//* modal, opan, close
+const backgroundModal = document.querySelector('.backgroundModal')
+const item = document.getElementsByClassName('items')
+Array.from(item, el => el.addEventListener('click', function () {
+    const id = this.getAttribute('item-id');
+    for (const i of items) {
+        if (i.id == id) {
+            createModal(i.imgUrl, i.name, i.orderInfo.reviews, i.color, i.os, i.size.height, i.size.weight, i.size.depth, i.size.weight, i.chip.name, i.orderInfo.inStock, i.price)
+        }
+    }
+    console.log(backgroundModal)
+    backgroundModal.classList.add('open')
+}))
+backgroundModal.addEventListener('click', function () {
+    backgroundModal.classList.remove('open')
+})
+
+
+//* create modal
+function createModal(imgUrl, name, reviews, color, operating, height, width, depth, weight, chip, left, price) {
+    const picture = document.getElementsByClassName('modal__img-picture')
+    picture[0].src = `/img/${imgUrl}`
+    const productName = document.getElementsByClassName('modal__name')
+    productName[0].innerText = name
+    const positiveReviews = document.getElementsByClassName('positive')
+    positiveReviews[0].innerText = reviews
+    document.getElementById('color').innerText = color.join(', ')
+    document.getElementById('operating').innerText = operating
+    document.getElementById('height').innerText = height
+    document.getElementById('width').innerText = width
+    document.getElementById('depth').innerText = depth
+    document.getElementById('weight').innerText = weight
+    document.getElementById('chip').innerText = chip
+    document.getElementById('left').innerText = left
+    const modalPrice = document.getElementsByClassName('modal__price')
+    modalPrice[0].innerText = `$ ${price}`
```

Comment:
modalPrice[0].innerText = `${price}`

Reply:
Hello )
here, before the price variable, you need to indicate that the price is in dollars.